core/src/main/scala/org/apache/spark/scheduler/SchedulableBuilder.scala

```diff
@@ -59,12 +59,15 @@ private[spark] class FairSchedulableBuilder(val rootPool: Pool, conf: SparkConf)
   val FAIR_SCHEDULER_PROPERTIES = "spark.scheduler.pool"
   val DEFAULT_POOL_NAME = "default"
   val MINIMUM_SHARES_PROPERTY = "minShare"
+  val MAXIMUM_SHARES_PROPERTY = "maxRunningTasks"
   val SCHEDULING_MODE_PROPERTY = "schedulingMode"
+  val PARENT_PROPERTY = "parent"
   val WEIGHT_PROPERTY = "weight"
   val POOL_NAME_PROPERTY = "@name"
   val POOLS_PROPERTY = "pool"
   val DEFAULT_SCHEDULING_MODE = SchedulingMode.FIFO
   val DEFAULT_MINIMUM_SHARE = 0
+  val DEFAULT_MAXIMUM_SHARE = Int.MaxValue
```

Comment:
I think this should be MAXIMUM_TASKS? (and same with the MAXIMUM_SHARES_PROPERTY)